app/services/api/v3/ecf/transfers_query.rb

```diff
@@ -14,6 +14,7 @@ def users
 
           scope = User
             .includes(user_includes)
+            .select("users.*", "induction_records.created_at")
```

Comment:
discussed in person: we need to ensure distinct runs on ids and not on created at